src/osu_put_latency.jl

```diff
@@ -0,0 +1,82 @@
+export OSUPutLatency
+
+struct OSUPutLatency <: MPIBenchmark
+    conf::Configuration
+    name::String
+end
+
+function OSUPutLatency(T::Type=Float32;
+                     filename::Union{String,Nothing}="julia_osu_put_latency.csv",
+                     kwargs...,
+                     )
+    return OSUPutLatency(
+        Configuration(T; filename, max_size=2 ^ 20, kwargs...),
+        "OSU Put Latency",
+    )
+end
+
+function osu_put_latency(T::Type, bufsize::Int, iters::Int, comm::MPI.Comm, synchronization_option::String)
+    win = MPI.Win_create(ones(T, bufsize), comm)
+    if(synchronization_option == "lock")
+       return run_put_with_lock(T, bufsize, iters, comm, win)
+    elseif(synchronization_option == "fence")
+       return run_put_with_fence(T, bufsize, iters, comm, win)
+    end
+end
+
+function run_put_with_lock(T::Type, bufsize::Int, iters::Int, comm::MPI.Comm, win::MPI.Win)
+    rank = MPI.Comm_rank(comm)
+    buffer = ones(T, bufsize)
+    root = 0
+    timer = 0.0    
+    tic = 0.0
+     
+    if rank == root
+        for i in 1:iters
+            if i == 1
+                tic = MPI.Wtime()
+            end
```

Comment:
```suggestion
    tic = MPI.Wtime()

    if rank == root
        for i in 1:iters
```
Do you need a barrier before `tic`?